Properties/Resources.fr-FR.resx

```diff
@@ -0,0 +1,303 @@
+  </data>
+  <data name="SelectAtLeast" xml:space="preserve">
+    <value>Vous devez sélectionner au moins un jeu.</value>
+  </data>
+  <data name="UploadingFes1" xml:space="preserve">
+    <value>Transfert et exécution de fes1...</value>
+  </data>
+  <data name="UploadingGames" xml:space="preserve">
+    <value>Transfert des jeux...</value>
+  </data>
+  <data name="UploadingKernel" xml:space="preserve">
+    <value>Transfert du Kernel...</value>
+  </data>
+  <data name="VerifyFailed" xml:space="preserve">
+    <value>La vérification a échouée :O Ca n'est pas bon signe.</value>
+  </data>
+  <data name="Verifying" xml:space="preserve">
+    <value>Vérification...</value>
+  </data>
+  <data name="WaitingForDevice" xml:space="preserve">
+    <value>En attente du périphérique...</value>
+  </data>
+  <data name="Warning" xml:space="preserve">
+    <value>Attention!</value>
+  </data>
+  <data name="Wow" xml:space="preserve">
+    <value>Wow</value>
+  </data>
+  <data name="ManyGames" xml:space="preserve">
+    <value>La NES Mini ne peut contenir que 97 jeux mais vous avez sélectionné {0} jeux. Voulez-vous continuer?</value>
+  </data>
+  <data name="PleaseTryAgainUSB" xml:space="preserve">
+    <value>Si ce problème se répète, essayez d'utiliser un autre port USB ou un autre cable USB.</value>
+  </data>
+  <data name="XpZadig" xml:space="preserve">
+    <value>Merci d'utiliser le programme Zadig pour installer le driver sur Windows XP/2000.</value>
+  </data>
+  <data name="Donate" xml:space="preserve">
+    <value>P.S. Si vous aimez l'application vous pouvez me faire un don. Vous trouverez mon PayPal dans la fenêtre "À propos".</value>
+  </data>
+  <data name="FirstRun" xml:space="preserve">
+    <value>Bonjour! Je suis content que vous utilisiez mon application. Elle est simple d'utilisation. Cliquez simplement sur "Ajouter plus de jeux", choisissez vos ROMs, cliquez sur "Synchroniser" et suivez les instructions. Bonne chance!</value>
+  </data>
+  <data name="Hello" xml:space="preserve">
+    <value>Bonjour</value>
+  </data>
+  <data name="KernelDumped" xml:space="preserve">
+    <value>Votre Kernel Original est sauvegardé dans le dossier "dump". Il est judicieux de ne pas le perdre. </value>
+  </data>
+  <data name="DeletePreset" xml:space="preserve">
+    <value>Voulez-vous supprimer le preset "{0}"?</value>
+  </data>
+  <data name="FourScreenNotSupported" xml:space="preserve">
+    <value>Désolé, {0} utilise le mode quatre-écran (note:  Dans de rare cas le jeu a de la mémoire VRAM additionel dans la cartouche) et cette fonctionnalité n'est pas supportée par la NES Mini. Le jeu ne démarrera sûrement pas. Voulez-vous quand même l'ajouter?</value>
+  </data>
+  <data name="InputPreset" xml:space="preserve">
+    <value>Entrez le nom du Preset pour le jeu sélectionné.</value>
+  </data>
+  <data name="NewPreset" xml:space="preserve">
+    <value>Nouvel sélection de Preset</value>
```

Comment:
I don't have this one in context, but that could be:
`Nouvelle sélection de Preset` or `Sélection d'un nouveau Preset`